contributions/presentation/week3/gchang-wska/README.md

```diff
@@ -0,0 +1,20 @@
+# Presentation Proposal
+
+## Members
+Gabriel Chang (gchang@kth.se)
+Github username: ChangGabriel
+
+William Skagerström (wska@kth.se)
+Github username: wska
+
+## Topic
+An introduction to code coverage
+
+## Outline
+* What is code coverage
+* How it is used in devops
```

Comment:
Can you elaborate on this? Do you mean what tools are used? Do you mean you want to discuss how developers decide what coverage metrics they should use? Or ...?